packages/app/src/components/applicationStages/ApplicationStages.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useContext, useEffect } from 'react';
+import { ActiveNominationContext } from '../../utils/context/ActiveNominationContext';
+import nominationsAPI from '../../utils/API/nominationsAPI';
+import './style.css';
+
+const ApplicationStages = () => {
+  const [activeNomination, setActiveNomination] = useContext(
+    ActiveNominationContext
+  );
+  const [currentStatus, setCurrentStatus] = useState('received');
+  const status = [
+    'Received',
```

Comment:
The strings for these statuses are repeated in a couple of different places, and they're always expected to match. This could cause annoying bugs if, for example, you were using `'Awaiting HIPAA'` in one place and `'Awaiting HIPPA'` in another.

Can you think of some way to fix this, so we don't have to repeat the same string in multiple places?

Reply:
How did this get resolved?